source/declarations.tex

```diff
@@ -4239,3 +4240,414 @@
 could have the same address as \tcode{buckets}
 if their respective types are all empty.
 \end{example}
+
+\rSec2[dcl.attr.contract]{Contract attributes}%
```

Comment:
We keep the subclauses of [dcl.attr] in alphabetical order. (So far that's been by attribute name, but the contract attributes throw a spanner in the works here.)

I'm not sure what the best answer is here.

Reply:
"We keep the subclauses of [dcl.attr] in alphabetical order."
Ah, that wasn't clear to me. But yes, the current list looks alphabetical.

Sort according to "contract" maybe? We already sort "likely" and "unlikely" under "likelihood".
@zygoloid, awaiting your answer before actually doing the move.

Reply:
Sure, let's sort by heading name.

Reply:
Fixed.